packages/DataGrid/src/DataGrid.js

```diff
@@ -38,7 +41,10 @@ const propTypes = {
   onPressShiftSpaceBar: PropTypes.func,
   /** Callback when Spacebar is pressed */
   onPressSpaceBar: PropTypes.func,
+  /** Callback when user click the f key. Might change in the future */
```

Comment:
I didn't know we use the "f" key. What is this for?

Reply:
sarah made the decision is to fire the keyboard event to select a row :)